src/content.ts

```diff
@@ -982,6 +982,23 @@ function previewTime(time: number, unpause = true) {
     }
 }
 
+//send telemetry and count skip
+function sendTelemetryAndCount(skippingSegments: SponsorTime[], secondsSkipped: number, fullSkip: boolean) {
```

Comment:
Might be good to add a `Config.config.trackViewCount` check here just to be safe

Reply:
Oh, yes forgot about that one